### ISSUE TITLE
Investigate Statistics Canada (StatCan) administrative data

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#666]
The Statistics Canada website has several administrative datasets that should be compared to existing records in Who's On First, including, but not limited to:

```
Provinces/territories
Federal electoral districts
Economic regions
Census divisions
Aggregate dissemination areas
Census metropolitan areas
Lakes and rivers (polygons)
Coastal waters (polygons)
```

**Census divisions**: [link](http://www5.statcan.gc.ca/access_acces/alternative_alternatif.action?l=eng&dispext=zip&teng=lcd_000b16a_e.zip&k=%20%20%20%2032286&loc=http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/files-fichiers/2016/lcd_000b16a_e.zip)
**Other administrative data**: [link](http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/bound-limit-2016-eng.cfm)
**Data license**: [link](http://www.statcan.gc.ca/eng/reference/licence)

____________________________
Province-level shapes from StatCan:
<img width="509" alt="screen shot 2017-02-22 at 3 11 50 pm" src="https://cloud.githubusercontent.com/assets/18567700/23237284/863edd92-f911-11e6-93f9-af43db05b7d9.png">

Province-level shapes in WOF:
<img width="443" alt="screen shot 2017-02-22 at 3 13 08 pm" src="https://cloud.githubusercontent.com/assets/18567700/23237297/94d5d45a-f911-11e6-89d0-cf45499eef69.png">

**Comment by burritojustice** [comment]
note that in most if not all provinces, `census divisions` are the equivalent of `counties` (which really don't exist by that name in Canada, except for a few eastern provinces)

**Comment by nvkelso** [comment]
Let's hold off on this issue until the final mesoshapes sprint, which includes Canada.

**Comment by robertoamoreno** [comment]
This might help http://www12.statcan.gc.ca/census-recensement/2011/geo/bound-limit/bound-limit-2016-eng.cfm and 
http://www12.statcan.gc.ca/census-recensement/2016/geo/ADA/adainfo-eng.cfm

**Comment by nvkelso** [comment]
Counties are tracked here: https://github.com/whosonfirst-data/whosonfirst-data/issues/926.

**Comment by stepps00** [comment]
Note: deprecate bunk county records like: https://whosonfirst.mapzen.com/spelunker/id/890456525/ after all StatCan data is in a PR.

**Comment by stepps00** [comment]
Closed, as #1235 was merged -- but we can reopen if we decide on harvesting other admin data from StatsCan.
